generators/generator-bot-calendar/generators/app/templates/README.md

```diff
@@ -10,25 +10,46 @@ In order to test this bot locally, you will need the following services provisio
 - Azure Bot Registration
 - Language Understanding (LUIS)
 
+#### Configure Microsoft App Password
+1. Open your Azure Bot Channels Regisration in the Azure Portal
+2. In the **Configuration** tab, click **Manage** next to your Microsoft App ID
+3. In the Certificates & secrets tab, click **New client secret**
+4. Assign a name and an expiration period, then click **Add**
+5. Copy the secret value and save for later use along with your Microsoft App ID
+
 ## Configure Authentication
-You must configure an authentication connection on your Azure Bot Registration in order to log in and access Microsoft Graph resources. 
-
-### Using Azure Portal
-* Open your **Azure Bot Service** resource and go to the **Settings** tab
-* Under **OAuth Connection Settings**, click **Add setting**
-* Configure select **Azure Active Directory v2** from the provider dropdown then fill out the fields with the following:
-  * Client Id: Your bot App Id
-  * Client secret: Your bot App Password
-  * Tenant: common
-  * Scopes: **Calendars.ReadWrite Contacts.Read People.Read User.ReadBasic.All** 
-* In the **Settings** tab, click **Manage** next to your Microsoft App ID
-* In the API permissions tab, add the following scopes: **User.ReadBasic.All Calendars.ReadWrite Contacts.Read People.Read**
-* Add the following property in Bot Settings in composer:
-  ```
-  "oauthConnectionName": "[your connection name]"
-  ```
-
-### Using [Azure CLI]()
+You must configure an authentication connection on your Azure Bot Registration in order to log in and access Microsoft Graph resources. You can configure these settings either through the Azure Portal or via the Azure CLI.
+
+### Option 1: Using the Azure Portal
+1. Open your **Bot Channels Registration** resource and go to the **Configuration** tab
+2. Click **Add OAuth Connection Settings**
+3. Assign your connection setting a name (save this value for later)
+4. Select **Azure Active Directory v2** from the Service Provider dropdown.
+5. Fill in the following fields and click **Save**:
+    * **Client id**: your Bot App Id
+    * **Client secret**: your Bot App Password
+    * **Tenant ID**: your Azure Active Directory tenant ID, or "common" to support any tenant
+    * **Scopes**: Calendars.ReadWrite Contacts.Read People.Read User.ReadBasic.All
+6. In the **Configuration** tab, click **Manage** next to your Microsoft App ID
+7. In the API permissions tab, click **Add a permission**
+8. Click **Microsoft Graph** and add the following scopes: 
+    * Calendars.ReadWrite
+    * Contacts.Read
+    * People.Read
+    * User.ReadBasic.All
+
+9. In the Authentication tab, click **Add a platform**
+    1. Select **Web**
+    2. Set the URL to https://token.botframework.com/.auth/web/redirect.
+10. In Bot Framework Composer, open your **Project Settings** and toggle the **Advanced Settings View**
+1. Set the following property to the value from Step 3:
+    ```
+    {
+      "oauthConnectionName": "Outlook",
```

Comment:
You may want to specify where in the json to add it to, cause the object is nested you might want to specifically direct them to it. (Maybe include a couple other lines from the same level in here too as a hint of some sort, like "MicrosoftAppId")

Reply:
We may also want to tell them if they don't have the setting configured correctly what error they should expect. (see my PR for the emit trace for example)

Reply:
We should really start thinking about frequently encountered problems and start documenting them here as they come in.

Reply:
The setting already exists in the templates, its just blank. I can't consistently show where it will show in the appsettings file since it gets added by the generator